db/migrate/20180405145642_remove_provider_region_for_google_provider.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveProviderRegionForGoogleProvider < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Removing provider_region from ManageIQ::Providers::Google::CloudManager") do
+      ExtManagementSystem.where(:type => 'ManageIQ::Providers::Google::CloudManager').each do |provider|
```

Comment:
@ZitaNemeckova 
I think that we do it in one query.
```ruby
ExtManagementSystem.where(:type => 'ManageIQ::Providers::Google::CloudManager').update_all(:provider_region => nil)
```

Reply:
@lpichler I did but miq-bot complained about `update_all`. I don't mind changing it back if miq-bot's complain isn't important.